app/settings/tokens/edit/controller.ts

```diff
@@ -22,10 +22,9 @@ export default class SettingsTokensEditController extends Controller {
 
     @action
     refresh() {
-        this.clearTokenValue();
-
-        // Send action to route
-        this.send('refreshRoute');
+        if (this.token && this.token.tokenValue) {
+            this.token.tokenValue = undefined;
+        }
     }
 
     clearTokenValue() {
```

Comment:
Do we still need this?

Reply:
I have now persisted the cleared value with save() after clearing the value. clearTokenValue() was unloading the record from the Ember store, which may have resulted in some of the errors seen. In the documentation, it states that this causes the record to be destroyed, which is not quite what we want here.

Reply:
Calling save() on clicking the Edit scopes button will make a PATCH request to the BE, which shouldn't be necessary, since at that point nothing should be updated yet. Unloading the record from the Ember store and reloading it from the BE should be the way to go.

Reply:
@adlius the `unloadRecord()` call is still causing issues when called. This fix however will persist the changes for the cleared token value and render the token with the appropriate values populated when the edit button is clicked. It will now also work with the delete and save buttons.